frame/proxy/src/lib.rs

```diff
@@ -476,13 +459,21 @@ decl_module! {
 		/// - A: the number of announcements made.
 		/// - P: the number of proxies the user has.
 		/// # </weight>
-		#[weight = T::WeightInfo::reject_announcement(T::MaxPending::get(), T::MaxProxies::get().into())]
-		fn reject_announcement(origin, delegate: T::AccountId, call_hash: CallHashOf<T>) {
+		#[pallet::weight(
+			T::WeightInfo::reject_announcement(T::MaxPending::get(), T::MaxProxies::get().into())
+		)]
+		pub(super) fn reject_announcement(
+			origin: OriginFor<T>,
+			delegate: T::AccountId,
+			call_hash: CallHashOf<T>
+		) -> DispatchResultWithPostInfo {
 			let who = ensure_signed(origin)?;
 			Self::edit_announcements(&delegate, |ann| ann.real != who || ann.call_hash != call_hash)?;
+
+			Ok(().into())
 		}
 
-		/// Dispatch the given `call` from an account that the sender is authorised for through
+		/// Dispatch the given `call` from an account that the sender is authorized for through
```

Comment:
I think we have a preference for british spelling. ;)

Reply:
TIL

Reply:
def no, we should stick with US spelling where possible, as this is standard for Computer Science.

Any UK spelling should ideally be corrected.

Reply:
see https://github.com/paritytech/substrate/pull/4940